docs/meshstack.mesh-policies.md

```diff
@@ -0,0 +1,38 @@
+---
+id: meshstack.mesh-policies
+title: Mesh Policies
+---
+
+meshPolicies can be set up between meshObjects, i.e., meshCustomer, meshProject, meshLandingZone and
+meshCustomerUserGroup. In the administration area, there is an option of 'Policies' in the 'Compliance' section of the
+navigation area. After clicking policies, you can view all policies in the system. Since this is a BETA version we only
```

Comment:
I dont get the reasoning behind the beta. The described behaviour will not change

Reply:
Then why we are showing BETA in the panel. I thought in the future we can define policies based on other fields too including tags.